### DISCUSSION TITLE
MicroProfile Compatibility

### DISCUSSION THREAD

**Discussion by geziefer** [quarkusio/quarkus#36597] [Community]
One of the features which made Quarkus special for me among other microservice-oriented runtimes was the support for MicroProfile, since I absolutely support standards.
Why is it, that this stopped at version 4.1 of MicroProfile (current version being 6.1, 7 coming soon)?

**Comment by maxandersen** [comment]
Quarkus supports Eclipse microprofile 6.

Where do you see we stoped at 4.1?

**Reply by geziefer** [reply]
Well at least Quarkus is last mentioned on 4.1 compatibility page (https://microprofile.io/compatible/4-1/#). If that was an up-to-dateness issue of MicroProfile web site, Quarkus project should tell them. Additionally, I was not able to find a version compatibility on Quarkus web site, only references to specific APIs (https://quarkus.io/standards/). If that adds up to a complete and higher version, it should mention it there.
